launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/LDClient.java

```diff
@@ -211,22 +219,31 @@ public void onError(Throwable e) {
      * @param context          the initial evaluation context; see {@link LDClient} for more
      *                         information about setting the context and optionally requesting a
      *                         unique key for it
-     * @param startWaitSeconds maximum number of seconds to wait for the client to initialize
+     * @param startWaitSeconds maximum number of seconds to wait for the client to initialize.  Determines when this
+     *                         function will return if no flags have been fetched.  If you use a large timeout, then
+     *                         any network delays could cause your application to wait a long time before continuing
+     *                         execution.
+     *
      * @return the primary LDClient instance
      * @see #init(Application, LDConfig, LDContext)
      * @since 3.0.0
      */
     public static LDClient init(Application application, LDConfig config, LDContext context, int startWaitSeconds) {
         initSharedLogger(config);
         getSharedLogger().info("Initializing Client and waiting up to {} for initialization to complete", startWaitSeconds);
+        if (startWaitSeconds >= EXCESSIVE_INIT_WAIT_SECONDS) {
+            getSharedLogger().warn("LDClient.init called with high start wait time parameter of {} seconds.", startWaitSeconds);
```

Comment:
I just remembered I was going to update the spec for this.

Something like:
`LDClient.init was called with a start wait time parameter of greater than {}. We recommend a timeout of less than {}.`

So that it contains actionable data.

Reply:
I matched iOS here which has merged.  I agree with your comment that it helps the customer more.  What is the recommended timeout for that second parameter?

Reply:
They are both the same number. The EXCESSIVE_INIT_WAIT_SECONDS

Reply:
Done